### PR TITLE
EES2 Project News

### PR DESCRIPTION
**Pull request by t-a-bonnet**
Added a 'Project News' section to the EES2 project page and created a news item.

### CODE REVIEW

**Comment by azaroth42** [comment]
Can we rephrase it to be "outcomes" rather than "news"? News implies that it's, well, new ... and before long it will not be new. Old news makes things look like there's no activity, which isn't the case either for the project or linked art more generally.

**Comment by t-a-bonnet** [comment]
Yes, that makes sense. Let's go with 'outcomes'.
